backend/app/core/exceptions/base.py

```
@@ -1,8 +1,20 @@
+from fastapi import HTTPException
+
+
 class AppHTTPException(HTTPException):
-    def __init__(self, status_code: int, message: str, error_code: str, details: Any = None, headers: Optional[Dict[str, str]] = None):
-        self.error_code = error_code
+    def __init__(
+        self,
+        status_code: int,
+        message: str,
+        details: Any = None,
+        title: str = None,
```

Ruff (0.14.13)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)